src/Microsoft.AspNetCore.Blazor.Browser/Services/BrowserServiceProvider.cs

```diff
@@ -43,7 +44,10 @@ private void AddDefaultServices(ServiceCollection serviceCollection)
         {
             var uriHelper = new BrowserUriHelper();
             serviceCollection.AddSingleton<IUriHelper>(uriHelper);
-            serviceCollection.AddSingleton(new HttpClient(uriHelper));
+            serviceCollection.AddSingleton(new HttpClient(new BrowserHttpMessageHandler())
```

Comment:
Will the default `HttpMessageHandler` work?

Reply:
No, Mono will throw `PlatformNotSupportedException`. We're hoping to change Mono-for-WebAssembly's default handler to the `BrowserHttpMessageHandler` at some point.

Reply:
Okay, thanks.